specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -119,6 +123,30 @@ export const notificationRenderers: IR<
       />
     );
   },
+  'record-merge-starting'() {
+    return mergingText.mergingHasStarted();
+  },
+  'record-merge-failed'() {
+    return mergingText.mergingFailed();
+  },
```

Comment:
(optional) Preferably, we would include more information for these two processes. Namely, an identifier that the user can use to keep track of which merging process is which. This can be something like the table and id of the merged record, or the celery task id itself. 

There can be multiple merges ongoing at the same time, and when the user just receives "Your merging process failed", it can be impossible to differentiate _which_ one failed.

Reply:
(optional) 
On the topic of providing more information, I would love to have a link to the Merge Records dialog from the `record-merge-failed` notification. 
You should be able to construct the URL using the ids of the records (which is included in the payload). 
(The url is in the format `http://localhost/specify/overlay/merge/Agent/?records=9,10,11`, where records is an "array" of the ids)

See https://github.com/specify/specify7/blob/3b6a6ab6593bbb790a05a90d58bf35d2b484e51a/specifyweb/frontend/js_src/lib/components/Merging/index.tsx#L88-L90C12

Also, including the reason might be beneficial for debugging purposes. At the very least, I would hide the reason under a "dropdown" (details). 
If that is still too far, we should have the reason when the frontend is in DEBUG mode